src/framework/ui/topNavigation/topNavigation.component.tsx

```diff
@@ -135,12 +131,26 @@ export class TopNavigationComponent extends React.Component<TopNavigationProps>
 
   static styledComponentName: string = 'TopNavigation';
 
+  private getAlignmentDependentStyles = (): StyleType | null => {
```

Comment:
can be refactored to stylesheet function?
e.g styles.container(alignment)

Reply:
It seems to me that this solution is a little more obvious to read.